cmake.deps/CMakeLists.txt

```
@@ -17,21 +17,13 @@ include(Deps)
 include(Find)
 include(Util)
 
-set_default_buildtype(Release)
-get_property(isMultiConfig GLOBAL PROPERTY GENERATOR_IS_MULTI_CONFIG)
-if(NOT isMultiConfig)
-  list(APPEND DEPS_CMAKE_ARGS -D CMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE})
-endif()
+#-------------------------------------------------------------------------------
+# User settings
+#-------------------------------------------------------------------------------
 
-set(DEFAULT_MAKE_CFLAGS CFLAGS+=-g)
-
-check_c_compiler_flag(-Og HAS_OG_FLAG)
-if(HAS_OG_FLAG)
-  set(DEFAULT_MAKE_CFLAGS CFLAGS+=-Og ${DEFAULT_MAKE_CFLAGS})
-endif()
-
-set(DEPS_INCLUDE_FLAGS "-I${DEPS_INSTALL_DIR}/include -I${DEPS_INSTALL_DIR}/include/luajit-2.1")
+set(IGNORE_SHA FALSE)
```

seems like a `DEPS_` prefix makes sense. We already have `DEPS_BUILD_DIR`, etc.
```suggestion
set(DEPS_IGNORE_SHA FALSE)
```